frontends/web/src/routes/settings/manage-account-guide.tsx

```diff
@@ -17,17 +17,36 @@
 import { useTranslation } from 'react-i18next';
 import { Entry } from '../../components/guide/entry';
 import { Guide } from '../../components/guide/guide';
+import { IAccount } from '../../api/account';
+import { isBitcoinOnly } from '../account/utils';
 
-export const AccountGuide = () => {
+type TAccountGuide = {
+  accounts: IAccount[]
+}
+
+export const AccountGuide = ({ accounts }: TAccountGuide) => {
   const { t } = useTranslation();
+  const hasOnlyBTCAccounts = accounts.every(({ coinCode }) => isBitcoinOnly(coinCode));
   return (
     <Guide>
       <Entry key="whatAreAccounts" entry={t('guide.accounts.whatAreAccounts')} />
       <Entry key="whyIsThisUseful" entry={t('guide.accounts.whyIsThisUseful')} />
       <Entry key="whatIsRememberWallet" entry={t('guide.accounts.whatIsRememberWallet')} />
       <Entry key="recoverAccounts" entry={t('guide.accounts.recoverAccounts')} />
       <Entry key="moveFunds" entry={t('guide.accounts.moveFunds')} />
-      <Entry key="howtoAddTokens" entry={t('guide.accounts.howtoAddTokens')} />
+      { !hasOnlyBTCAccounts && (
+        <>
+          <Entry key="supportedCoins" entry={{
+            link: {
+              text: t('guide.accounts.supportedCoins.link.text'),
+              url: 'https://bitbox.swiss/coins/',
```

Comment:
Would be nice if the links are language aware `i18n.resolvedLanguage` and point to https://bitbox.swiss/de/coins/ or https://bitbox.swiss/es/monedas/ with fallback to the English page.

similar to

https://github.com/digitalbitbox/bitbox-wallet-app/blob/c8e3255951afa6671345f424f68ddbfb4adb6717/frontends/web/src/routes/device/bitbox02/settings-guide.tsx#L22-L31